### PR TITLE
Fix purge for tailwind DAH-856

### PR DESCRIPTION
**Pull request by software-project**
Resolves [DAH-856](https://sfgovdt.jira.com/browse/DAH-856)

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -7,7 +8,8 @@ const bloomTheme = cloneDeep(require("@bloom-housing/ui-components/tailwind.conf
 
 // tailwind will automatically purge unused styles when `NODE_ENV` is set to `production`
 bloomTheme.purge = [
-  __dirname + '/app/javascript/**/*.tsx'
+  "app/javascript/**/*.tsx",
+  "node_modules/@bloom-housing/ui-components/src/**/*.tsx",
```
Comment (akegan) [inline tailwind.config.js:12]: @software-project can you explain how this works?

**Comment by akegan** [comment]
@software-project tests are failing FYI

File: tailwind.config.js (hunk 1 above)
Comment (software-project) [reply tailwind.config.js:13]: Done

**Review by akegan — APPROVED** [review]
lgtm including QA. Just kicked off the legacy e2e tests to try to get them to pass. once they pass we should be good to merge! 🎉

**Pull request merged**